pkg/master/controller.go

```diff
@@ -204,6 +208,7 @@ func (c *Controller) SetEndpoints(serviceName string, ip net.IP, port int) error
 	// subset, one port, N IP addresses).
 	formatCorrect, ipCorrect := checkEndpointSubsetFormat(e, ip.String(), port, c.MasterCount)
 	if !formatCorrect {
+		glog.Warningf("Old endpoints need fixing %v %v", serviceName, e)
```

Comment:
This error message is not needed. This is a case that's possible to get to legitimately.